### DISCUSSION TITLE
Defining groups.

### DISCUSSION THREAD

**Discussion by ivandatasci** [Valdes-Tresanco-MS/gmx_MMPBSA#542] [General]
Hello everyone.

Could I have some guidance to address a simple question?

I intend to compute the energy of binding in a heterotrimeric complex.
The three peptide chains are named H, L, and X.

My "protein" is the heterodimer of peptide chains H and L.
My ligand is peptide chain X.

How do I indicate that grouping?

I am attaching my .pdb in case it helps. 

Thank you.

Ivan
[complex.pdb.gz](https://github.com/user-attachments/files/17580041/complex.pdb.gz)

**Comment by Valdes-Tresanco-MS** [comment]
In gmx make_ndx:
1. use `splitch` to split the `(1) Protein` group into chains. For example `splitch 1`. This will generate an additional 3 groups.
2. Using `|` select H and L. For example, `20 | 21`. This will generate a new group.
3. Save and quit

Then use those groups in gmx_MMPBSA. 
Note: Use the tpr file otherwise you can't split the chains. For example, `gmx make_ndx -f md.tpr -n index.ndx`
